samples/attested_tls/server/enc/mbedtls/ecalls.cpp

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 #include <openenclave/enclave.h>
-#include "../../common/tls_server_enc_pubkey.h"
+#include "../../../common/tls_server_enc_pubkey.h"
```

Comment:
this ecalls.cpp file is another example of a file that doesn't seem to contain any mbedtls specific content.  Can this be left in enc?